.github/ISSUE_TEMPLATE/improvement.yaml

```
@@ -0,0 +1,33 @@
+name: Improvement
+title: "[Improvement] "
+description: Suggest an improvement on performance, code quality, user experience, etc.
+labels: [ "improvement" ]
+body:
+
+- type: textarea
+  attributes:
+    label: What would you like to be improved?
+    description: Please describe the problem you see and why it needs improvement.
+    placeholder: >
+      Clearly explain the issue with performance, code quality, or user experience.
```

Is this equivalent?
```suggestion
    placeholder: |
      Clearly explain the issue with performance, code quality, or user experience.
```

I don't see `>` anywhere in the [docs for issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)

---

let me make a change

---

done updated and checked all files twice.